.gitignore

```diff
@@ -26,3 +26,5 @@ dist/
 local/
 gh-pages/
 searx.egg-info/
+
+.vscode/
```

Comment:
Please do not add any editor specific line to this file.

Reply:
OK. How do you suggest I stop my config from being pushed without manually staging each of my commits?

Reply:
> without manually staging each of my commits?

what should that tell me?!? Please learn git and SCM basics .. to me it seems you won't learn for what all this is good.

Reply:
> How do you suggest I stop my config from being pushed without manually staging each of my commits?

See https://stackoverflow.com/questions/5724455/can-i-make-a-user-specific-gitignore-file